packages/vuetify/src/components/VTreeview/VTreeviewNode.ts

```diff
@@ -214,8 +215,27 @@ const VTreeviewNode = baseMixins.extend<options>().extend({
           'v-treeview-node__toggle--open': this.isOpen,
           'v-treeview-node__toggle--loading': this.isLoading,
         },
+        props: { value: this.isOpen },
+        attrs: {
+          'aria-expanded': String(this.isOpen),
+          'aria-label': this.text,
```

Comment:
`aria-label` should be used when there is no visible description for the element anywhere on the page. Since there is a visible text (the node text), `aria-labbeledby` [should be used instead](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute).

For that every text node in the treeview will need a unique ID.

Reply:
The checkbox (if selectable is enabled) also needs `aria-labelledby`. Currently NVDA just reads "button" when you focus on it.